cdap-ui/app/styles/themes/cdap/mixins.less

```diff
@@ -55,3 +55,59 @@
   .border-radius(10px);
   &:before { line-height: @line-height; }
 }
+
+.dropdown-defaults {
+  + .dropdown-menu {
+    background-color: transparent;
+    border: 0;
+    margin: 0;
+    padding: 0;
+    .border-radius(8px);
+    > li {
+      &:first-child a { .border-radius(8px 0 0 0); }
+      &:last-child a {
+        .border-radius(0 0 8px 8px);
+        &:hover { .border-radius(0 0 8px 8px); }
+      }
+      &:only-child a { .border-radius(8px 0 8px 8px); }
+      > a {
+        padding: 10px;
+        cursor: pointer;
+      }
+      my-file-select {
```

Comment:
I recommend keeping these mixins outside of fileselect.less. More often than not, dropdown menus contain actions beyond the fileselect directive, or don't contain a fileselect at all. One option would be to account for the filleselect style rules separately, in fileselect.less, but this seems like a fragmented approach.